<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LearningWindow</class>
 <widget class="QMainWindow" name="LearningWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="assembleButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Assemble</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>530</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QLabel" name="caseLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>400</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string>Case</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="motherboardLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string>Motherboard</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="gpuLabel">
    <property name="geometry">
     <rect>
      <x>549</x>
      <y>29</y>
      <width>231</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>GPU</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cpuLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="memoryLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>220</y>
      <width>200</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Memory</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ramLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>300</y>
      <width>41</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>Ram</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>530</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previousButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>530</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stepByStepButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>530</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Step-By-Step</string>
    </property>
   </widget>
   <widget class="QLabel" name="stepByStepLabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>131</width>
      <height>200</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);
color: rgb(0, 0, 0);
border: 3px solid black</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click the arrow buttons to go to the next or previous step&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>stepByStepLabel</zorder>
   <zorder>assembleButton</zorder>
   <zorder>testButton</zorder>
   <zorder>caseLabel</zorder>
   <zorder>motherboardLabel</zorder>
   <zorder>gpuLabel</zorder>
   <zorder>cpuLabel</zorder>
   <zorder>memoryLabel</zorder>
   <zorder>ramLabel</zorder>
   <zorder>nextButton</zorder>
   <zorder>previousButton</zorder>
   <zorder>stepByStepButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
